facia-press/app/controllers/HealthCheck.scala

```diff
@@ -11,13 +12,17 @@ class HealthCheck(
   toolPressQueueWorker: ToolPressQueueWorker,
   val controllerComponents: ControllerComponents
 )
-  extends HealthCheckController with Results {
+  extends HealthCheckController with Results with Logging {
   val ConsecutiveProcessingErrorsThreshold = 5
   override def healthCheck(): Action[AnyContent] = Action.async{
     if (!toolPressQueueWorker.lastReceipt.exists(_.plusMinutes(1).isAfter(DateTime.now))) {
-      successful(Results.InternalServerError("Have not been able to retrieve a message from the tool queue for at least a minute"))
+      val msg = "Have not been able to retrieve a message from the tool queue for at least a minute"
```

Comment:
I don't have all the context for why this logging is useful but  is it worth enhancing this message with some time related values? Right now we state that 1 minute has passed but perhaps we could log exactly how long has passed since `lastReceipt.exists`?

Reply:
Great idea, @oliverlloyd, @tjmw -- I was just hastily ensuring that what was currently output was logged, but we can do better.